src/internal/executer.interfaces.ts

```diff
@@ -10,3 +10,12 @@ export interface IRequestExecuterData {
   body?: any;
   queryParams?: QueryParam[];
 }
+
+/**
+ * Maps an object into a list of QueryParam
+ * @param   o The object to be mapped
+ * @returns QueryParam A list of query params
+ */
+export function toQueryParams(o: object): QueryParam[] {
```

Comment:
It wasn't expected to find function in `.interfaces.ts` file :) Maybe there is a better place to move it?

Reply:
agreed, moved to another file